packages/domain/entities/package.json

```diff
@@ -1,4 +1,4 @@
 {
-  "type": "module",
-  "main": "../dist/entities/index.js"
```

Comment:
Maybe I misunderstood... but does `preconstruct` creates these fallback directories?

Reply:
It does, yes

Reply:
so we can delete them? Sorry I am confused... I would assume this file would be delete then or am I missing something?

Reply:
we can add them to gitignore but every time we run a build they are generated